config/locales/en.yml

```diff
@@ -224,6 +224,7 @@ en:
       tax_returns_attributes:
         certification_level: certification level
         is_hsa: is HSA
+      invalid_tax_status: The provided tax status is not valid.
```

Comment:
Should be something like "the new tax return state was not recognized" (or something less mechanical).